app/models/rubric_criterion.rb

```diff
@@ -34,31 +36,35 @@ def update_assigned_groups_count
     self.assigned_groups_count = result.uniq.length
   end
 
-  # Just a small effort here to remove magic numbers...
   RUBRIC_LEVELS = 5
   DEFAULT_MAX_MARK = 4
   MAX_LEVEL = RUBRIC_LEVELS - 1
-  DEFAULT_LEVELS = [
-    {'name' => I18n.t('rubric_criteria.defaults.level_0'),
-     'description' => I18n.t('rubric_criteria.defaults.description_0')},
-    {'name' => I18n.t('rubric_criteria.defaults.level_1'),
-     'description' => I18n.t('rubric_criteria.defaults.description_1')},
-    {'name' => I18n.t('rubric_criteria.defaults.level_2'),
-     'description' => I18n.t('rubric_criteria.defaults.description_2')},
-    {'name' => I18n.t('rubric_criteria.defaults.level_3'),
-     'description' => I18n.t('rubric_criteria.defaults.description_3')},
-    {'name' => I18n.t('rubric_criteria.defaults.level_4'),
-     'description' => I18n.t('rubric_criteria.defaults.description_4')}
-  ]
+
+  def levels
```

Comment:
Remove this---you can access `self.levels` already, just by having the `has_many` association defined above.